### ISSUE TITLE
Installing from ssh git repo fails

### ISSUE THREAD

**Issue by binarylegit** [bower/bower#2408]
**Output of `bower -v && npm -v && node -v`:**
```
1.8.0
3.8.9
v4.4.3
```

**Additional environment details (proxy, private registry, etc.):**
Windows 10

**Steps to reproduce the issue:**

1. Attempt to install a package from a git repository via ssh.  The following commands return the following results:
```
bower install git@bitbucket.org:team/repo.git -V
bower                        ENOTFOUND Package git not found
```

**Describe the results you expected:**
In version 1.7.9 this same command is successful and pulls down the dependency.  Further, according to the documentation it appears that this command should have worked.

**Additional information:**

**Comment by manufacturedba** [comment]
Having the same issue.  Maybe it has to do with `@` now being supported as a version delimiter?

**Comment by binarylegit** [comment]
It looks like this pull request may fix the issue:

https://github.com/bower/bower/pull/2395

**Comment by timcroydon** [comment]
Can confirm this also affects OSX.

**Comment by sheerun** [comment]
This should be fixed already as #2395 was merged
